llvm/lib/Analysis/MLInlineAdvisor.cpp

```diff
@@ -475,7 +485,7 @@ std::unique_ptr<InlineAdvice> MLInlineAdvisor::getAdviceImpl(CallBase &CB) {
   }
   // This one would have been set up to be right at the end.
   if (!InteractiveChannelBaseName.empty() && InteractiveIncludeDefault)
-    *ModelRunner->getTensor<int64_t>(getFeatureMap().size()) =
+    *ModelRunner->getTensor<int64_t>(getFeatureMap().size() - 1) =
```

Comment:
Trying to understand.. why did this change?

Reply:
Before, we were adding the extra feature to a local clone we'd send the model (see lines 84-85 on the left)